apps/console/src/features/claims/constants/claim-management-constants.ts

```diff
@@ -16,6 +16,8 @@
  * under the License.
  */
 
+import {SCIMConfigs} from "../../../extensions/configs/scim";
```

Comment:
Let's export `SCIMConfigs` from `../../../extensions/configs/index.ts`. so that we can shorten the import.
 
```suggestion
import { SCIMConfigs } from "../../../extensions/configs";
```

Reply:
Changed in commit 709db97832e822082dff6db0d23962b5e6773072